_sass/_base.scss

```diff
@@ -0,0 +1,206 @@
+/**
```

Comment:
Yes I know that.
I think to start, it'll be better to use pure CSS to reduce processes and the amount of knowledge to contribute.

I wanna keep it simple: A static pages builded after run a command, but nothing beyond it.